hermes-test-helper/src/main/java/pl/allegro/tech/hermes/test/helper/endpoint/BrokerOperations.java

```diff
@@ -39,8 +42,11 @@ public BrokerOperations(Map<String, String> kafkaZkConnection, ConfigFactory con
     public BrokerOperations(Map<String, String> kafkaZkConnection, int sessionTimeout, int connectionTimeout, String namespace) {
         zkClients = kafkaZkConnection.entrySet().stream()
                 .collect(toMap(Map.Entry::getKey,
-                               e -> new ZkClient(e.getValue(), sessionTimeout, connectionTimeout, ZKStringSerializer$.MODULE$)));
-
+                               e -> {
+                                   ZkConnection connection = new ZkConnection(e.getValue(), sessionTimeout);
+                                   ZkClient zkClient = new ZkClient(connection, connectionTimeout, ZKStringSerializer$.MODULE$);
+                                   return new ZkUtils(zkClient, connection, false);
```

Comment:
I will definitely iterate over this PR until it is stable on our test environment but we don't use kafka-security at the moment on any of our clusters yet. I can make it configurable but it would be hard for me to say that everything will go smoothly for you if you have kafka-security enabled.